compiler/luci/service/src/Nodes/CircleStridedSlice.test.cpp

```diff
@@ -41,3 +42,351 @@ TEST(CloneNodeTest, clone_StridedSlice)
   ASSERT_EQ(node_ss->new_axis_mask(), cloned_ss->new_axis_mask());
   ASSERT_EQ(node_ss->shrink_axis_mask(), cloned_ss->shrink_axis_mask());
 }
+
+TEST(ShapeRuleTest, strided_slice_static_shape)
+{
+  luci::CircleInput input;
+  luci::CircleConst begin;
+  luci::CircleConst end;
+  luci::CircleConst strides;
+
+  luci::CircleStridedSlice strided_slice;
+
+  input.shape({5, 6, 7, 8});
+  input.shape_status(luci::ShapeStatus::VALID);
+
+  begin.dtype(loco::DataType::S32);
+  begin.shape({4});
+  begin.size<loco::DataType::S32>(4);
```

Comment:
Do we need to set the size of tensor in CircleConst twice?
As far as I know, when we call the `shape()` or `size()` function, it eventually resizes the vector that it contains.

Reply:
As far as I know, shape manipulates rank and dim, while size manipulates the data of `CircleConst`.